guides/common/modules/proc_configuring-http-proxy-to-connect-to-cdn.adoc

```diff
@@ -1,29 +1,21 @@
 [id="configuring-http-proxy-to-connect-to-cdn_{context}"]
 = Configuring the HTTP Proxy to Connect to Red Hat CDN
 
-To configure subscription-manager with an HTTP proxy, follow the procedure below.
+.Prerequisites
+
+Your network gateway and the HTTP proxy must allow access to the following hosts:
+include::snip_host-names-for-http-proxy.adoc[]
+
+To configure the subscription-manager with the HTTP proxy, follow the procedure below.
 
 .Procedure
 
-. On the network gateway and the HTTP Proxy, enable TCP for the following host names:
-+
-[cols="2,1,1",options="header"]
-|====
-| Host name | Port | Protocol
-| subscription.rhsm.redhat.com | 443 | HTTPS
-| cdn.redhat.com |  443 | HTTPS
-| *.akamaiedge.net |  443 | HTTPS
-| cert.console.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
-| cert-api.access.redhat.com (if using Red{nbsp}Hat Insights) |  443 | HTTPS
-| api.access.redhat.com (if using Red{nbsp}Hat Insights) |  443 | HTTPS
-|====
-+
 {ProjectServer} uses SSL to communicate with the Red{nbsp}Hat CDN securely.
 Use of an SSL interception proxy interferes with this communication.
 These hosts must be whitelisted on the proxy.
-+
+
 For a list of IP addresses used by the Red{nbsp}Hat CDN (cdn.redhat.com), see the Knowledgebase article https://access.redhat.com/articles/1525183[Public CIDR Lists for Red{nbsp}Hat] on the Red{nbsp}Hat Customer Portal.
```

Comment:
If we're already moving things around, I'd move this text together with the table to the Prerequisites.